mappings/net/minecraft/enchantment/LoyaltyEnchant.mapping

```diff
@@ -0,0 +1 @@
+CLASS net/minecraft/class_3589 net/minecraft/enchantment/LoyaltyEnchant
```

Comment:
Should be renamed to `LoyaltyEnchantment` for consistency with the other enchantments.

```suggestion
CLASS net/minecraft/class_3589 net/minecraft/enchantment/LoyaltyEnchantment
```

Reply:
fixed in 0bb6d26